yaw_pid:
  ros__parameters:
    kp: 0.02
    ki: 0.0
    kd: 0.5
    dt: 0.5
    max: 0.1
    min: -0.1
    unwinding_factor: 3.0

    setpoint: 0.0
    is_axis_inverted: 1   # 1 or -1
    is_verbose: true